src/towncrier/test/test_settings.py

```diff
@@ -35,6 +37,53 @@ def test_base(self):
         self.assertEqual(config.underlines, ["=", "-", "~"])
         self.assertEqual(config.orphan_prefix, "~")
 
+    def test_markdown(self):
+        """
+        If the filename references an .md file and the builtin template doesn't have an
+        extension, add .md rather than .rst.
+        """
+        temp = self.mktemp()
+        os.makedirs(temp)
```

Comment:
Same as below, I believe this is not necessary?


```suggestion
```

Reply:
Ok, I removed this from the entire codebase for consistency